rocAL/rocAL/source/master_graph.cpp

```diff
@@ -316,6 +317,23 @@ MasterGraph::create_image(const ImageInfo &info, bool is_output)
     return output;
 }
 
+void
+MasterGraph::set_output(Image* output_image)
```

Comment:
I thought we have discussed to modify this into 2 separate routines

Reply:
We planned to make this change in the Tensor Branch Rajy